File: liband/src/main/java/com/liband/utils/DrawablesUtil.java
package com.liband.utils;

import android.graphics.drawable.GradientDrawable;
import android.view.View;

public class DrawablesUtil {

    public static void changeBackgroundDrawable(View view, int solidColor, int strokeColor, int strokeWidth, float radiusNumber) {

        GradientDrawable drawable = (GradientDrawable) view.getBackground();
        drawable.setColor(solidColor);
        drawable.setStroke(strokeWidth, strokeColor);
        drawable.setCornerRadius(radiusNumber);

    }

}

File: liband/src/main/java/com/liband/utils/DrawablesUtil.kt
package com.liband.utils

import android.graphics.drawable.GradientDrawable
import android.view.View

object DrawablesUtil {

    fun changeBackgroundDrawable(view: View, solidColor: Int, strokeColor: Int, strokeWidth: Int, radiusNumber: Float) {

        val drawable = view.background as GradientDrawable
        drawable.setColor(solidColor)
        drawable.setStroke(strokeWidth, strokeColor)
        drawable.cornerRadius = radiusNumber

    }

}
